inference-engine/src/gna_plugin/frontend/model_quantizer.hpp

```diff
@@ -128,12 +128,27 @@ class ModelQuantizer {
                     k--;
                 }
 
-                if ((infiniteLoopHistory.size() - i) % 2 == 0 && (infiniteLoopHistory.size() - i) / 2 == infiniteLoopHistory.size() - k) {
+                // for similar records, we will increase the pattern length to 4
```

Comment:
Why 4? Can we avoid magic numbers in code and/or add any explanations?

Reply:
Added  comments.